src/api-provider.ts

```diff
@@ -52,6 +52,16 @@ export class ApiProvider {
               prodParasPolkadotCommon.find((e) => e.info === "PolkadotAssetHub")
                 ?.providers || {}
             ).filter((e) => e.startsWith("wss://"));
+          } else if (chain === "bifrost_polkadot" || chain === "bifrost") {
+            const chainInfoName = "bifrost";
+            const haystack =
```

Comment:
took me a while :)

Reply:
> took me a while :)

Yeah. the ugly bit is that in `polkadot/apps-config` the `info` field is `bifrost` for both kusama as well as polkadot parachains. Unlike other parachains we're connecting with where the names are distinct. Hence the somewhat awkward way of handling it here.